main/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/bytebase/dbcost/client"
+	"github.com/bytebase/dbcost/client/aws"
+	"github.com/bytebase/dbcost/client/gcp"
+	"github.com/bytebase/dbcost/store"
+)
+
+var (
+	cloudProvider = make(map[store.CloudProvider]client.Client)
+	// renderEnvKey is set on render
+	renderEnvKey = "API_KEY_GCP"
+	apiKeyGCP    = ""
+	dirPath      = "./store/data"
+	fileName     = "rds.json"
+	filePath     = path.Join(dirPath, fileName)
+)
+
+func init() {
+	apiKeyGCP = os.Getenv(renderEnvKey)
```

Comment:
Why is this in init() instead of main()? Also why is the declaration of cloudProvider not in main().